daffodil-cli/bin.NOTICE

```diff
@@ -106,18 +113,11 @@ Scala Parser Combinators (lib/org.scala-lang.modules.scala-parser-combinators_2.
   LAMP/EPFL (https://lamp.epfl.ch/) and
   Lightbend, Inc. (https://www.lightbend.com/).
 
-Scala XML (lib/org.scala-lang.modules.scala-xml_2.12-1.1.0.jar)
+Scala XML (lib/org.scala-lang.modules.scala-xml_2.12-1.3.0.jar)
   scala-xml
   Copyright (c) 2002-2020 EPFL
   Copyright (c) 2011-2020 Lightbend, Inc.
 
   scala-xml includes software developed at
   LAMP/EPFL (https://lamp.epfl.ch/) and
   Lightbend, Inc. (https://www.lightbend.com/).
-
-Woodstox (lib/com.fasterxml.woodstox.woodstox-core-5.1.0.jar)
-  This product currently only contains code developed by authors
-  of specific components, as identified by the source code files.
-
-  Since product implements StAX API, it has dependencies to StAX API
-  classes.
```

Comment:
This appears to be a notice that was only applied to old version of woodstox. Newer versions do not include a notice that I can find.